### PR TITLE
ENH Join endpoint path segments

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-linkfield/issues/165

### CODE REVIEW

File: client/src/components/LinkField/LinkField.js [hunk 1]
```diff
@@ -2,16 +2,15 @@
 import React, { useState, useEffect, createContext } from 'react';
 import { bindActionCreators, compose } from 'redux';
 import { connect } from 'react-redux';
-import { injectGraphql } from 'lib/Injector';
```
Comment (emteknetnz) [inline client/src/components/LinkField/LinkField.js:5]: This is minor unrelated refactoring, I removed a couple of unused imports

File: client/src/components/LinkField/LinkField.js [hunk 2]
```diff
@@ -114,7 +113,7 @@ const LinkField = ({
    * Update the component when the 'Delete' button in the LinkPicker is clicked
    */
   const onDelete = (linkID) => {
-    let endpoint = `${Config.getSection(section).form.linkForm.deleteUrl}/${linkID}`;
+    let endpoint = joinUrlPaths(Config.getSection(section).form.linkForm.deleteUrl, linkID.toString());
```
Comment (emteknetnz) [inline client/src/components/LinkField/LinkField.js:116]: Need to convert the linkID int to a string otherwise joinURLPaths gets a weird array created when you pass in a string then an int 0 as arguments

**Review by GuySartorelli — APPROVED** [review]
Looks good, works well locally with both trailing slash config values. Can't find any that were missed.
